cmd/agent/app/reporter/grpc/reporter.go

```diff
@@ -55,14 +53,14 @@ func (r *Reporter) EmitZipkinBatch(zSpans []*zipkincore.Span) error {
 	if err != nil {
 		return err
 	}
-	return r.send(trace.Spans)
-}
-
-func (r *Reporter) send(spans []*model.Span) error {
 	var process model.Process
-	if len(spans) > 0 {
-		process = *spans[0].Process
+	if len(trace.GetSpans()) > 0 {
```

Comment:
nit: could make it nicer in Go; `if spans := trace.GetSpans(), len(spans) > 0 { use spans }`